modules/ebi-metagenomics/keggpathwayscompleteness/main.nf

```diff
@@ -37,9 +37,11 @@ process KEGGPATHWAYSCOMPLETENESS {
 
     give_completeness \\
         -i ${prefix}_aggregated_kos_per_contig.tsv \\
+        --add-per-contig \\
         -o ${prefix}
 
     mv ${prefix}/summary.kegg_pathways.tsv ${prefix}_summary_kegg_pathways.tsv
```

Comment:
give_completeness has https://github.com/EBI-Metagenomics/kegg-pathways-completeness-tool/blob/master/kegg_pathways_completeness/bin/give_completeness.py#L80
you can use that and no need to rename..

Reply:
Yeah, this one needs to be cleaned. I went back and forth with this one

Reply:
JIRA task so we don't forget about it https://www.ebi.ac.uk/panda/jira/browse/EMG-7787